src/lib/bootloader/stage1_device.rb

```diff
@@ -53,61 +56,66 @@ def underlaying_devices_for(dev)
       res.uniq
     end
 
+    def pv_disk(vg)
+      pvs = usable_pvs(vg)
+      pvs.map { |p| p.ancestors.find { |a| a.is?(:disk) }.name }
+    end
+
     # get one level of underlaying devices, so no recursion deeper
     def underlaying_devices_one_level(dev)
-      tm = Yast::Storage.GetTargetMap
-      disk_data = Yast::Storage.GetDiskPartition(dev)
-      if disk?(disk_data)
-        disk = Yast::Storage.GetDisk(tm, dev)
-        # md disk is just virtual device, so select underlaying device /boot partition
-        case disk["type"]
-        when :CT_MD then return underlaying_disk_with_boot_partition
-        when :CT_LVM
-          res = lvm_underlaying_devices(disk)
-          return res.map { |r| Yast::Storage.GetDiskPartition(r)["disk"] }
-        when :CT_DMRAID
-          return disk["devices"]
-        end
-      # given device is partition
-      else
-        part = Yast::Storage.GetPartition(tm, dev)
-        if part["type"] == :lvm
-          lvm_group = Yast::Storage.GetDisk(tm, disk_data["disk"])
-          return lvm_underlaying_devices(lvm_group)
-        elsif part["type"] == :sw_raid
-          return devices_on(part)
-        elsif part["fstype"] == Yast::Partitions.dmraid_name
-          mapper = Yast::Storage.GetDisk(tm, disk_data["disk"])
-          return mapper["devices"]
-        end
+      # FIXME: there is nothing like this right now in libstorage-ng
```

Comment:
Yes, please review the merge (like travis configuration, resulting changelog, etc.) but don't re-review the code. Merging is the important thing in this PR. The rest can be addressed, if you feel in fixing-stuff mood, in separate PRs. 😉